components/color-picker/index.jsx

```diff
@@ -240,14 +244,12 @@ class ColorPicker extends React.Component {
 		const workingColor = ColorUtils.getNewColor(
 			{
 				hex:
-					this.props.valueWorking ||
-					this.props.value ||
-					this.props.swatchColors[0],
+					this.props.valueWorking || this.props.value || '#000000'
```

Comment:
The working color should never be transparent, so I am intentionally setting black as default color if transparent is selected. Otherwise, we get in this kind of situation
![screen shot 2018-10-24 at 1 18 26 am](https://user-images.githubusercontent.com/1821787/47416527-c6e08000-d72a-11e8-8950-8ba5bc756e6e.png).